### ISSUE TITLE
`require-system-font-fallback` false positive in `@font-face` rule

### ISSUE THREAD

**Issue by Brainshaker95** [yuschick/stylelint-plugin-defensive-css#190]
**Describe the bug**  
`require-system-font-fallback` reports an error for `font-family` declarations inside `@font-face`. This is a false positive because fallback font stacks are not valid in that context.

**To Reproduce**  
1. Enable `defensive-css/require-system-font-fallback`  
2. Lint the following CSS:
  ```css
   @font-face {
     font-family: 'My Family';
   }
  ```
3. Observe the reported error

**Expected behavior**  
No error should be reported. `@font-face` defines a single font identifier, not a font stack, so fallback requirements should not apply.

**Additional context**  
The rule currently walks all `font` and `font-family` declarations without checking context. It should ignore declarations inside `@font-face`.

Thanks for providing this plugin!

**Comment by yuschick** [comment]
Hey 👋  This is a great catch that I had missed.

In the short term, you can use the `ignore` option to ignore the font-family name you're adding there so that it isn't flagged. In the mean time, I will put a proper fix in to account for actual `@font-face` declarations.

**Comment by Brainshaker95** [comment]
I have had a look at the rule and the fix seems quite straight forward. I can raise a PR if you want.

https://github.com/yuschick/stylelint-plugin-defensive-css/blob/0e518e15f9154ff744623872e04e7ad74f131fab/src/rules/require-system-font-fallback/rule.ts#L52-L55

->

```diff
    root.walkDecls(/^font(?:-family)?$/, (decl) => {
+      if (decl.parent?.type === 'atrule' && decl.parent.name === 'font-face') {
+        return;
+      }
+
      if (globalKeywords.includes(decl.value.trim())) {
        return;
      }
```

Additional test:
```ts
{
  code: '@font-face { font-family: "Custom Font"; }',
  description: '@font-face rule with custom font family',
}
```

**Comment by yuschick** [comment]
Thanks again for reporting. As you said, easy fix. This is now released as [v2.9.1](https://github.com/yuschick/stylelint-plugin-defensive-css/releases/tag/v2.9.1)

**Comment by Brainshaker95** [comment]
Thanks!
